core/src/main/java/org/elasticsearch/action/search/SearchRequestBuilder.java

```diff
@@ -123,14 +121,6 @@ public SearchRequestBuilder setTimeout(TimeValue timeout) {
     }
```

Comment:
why does the SearchRequestBuilder still has its own SearchSourceBuilder? Shouldn't it just set stuff to the inner SearchRequest?